package.json

```diff
@@ -334,20 +335,25 @@
                     "when": "view == extension.vsKubernetesExplorer && viewItem == vsKubernetes.resource.pod"
                 },
                 {
-                    "command": "extension.vsKubernetesTerminal",
+                    "command": "extension.vsKubernetesDeleteNow",
```

Comment:
Is there a reason to offer Delete Now only for pods and not for other resources?

Reply:
As the issue owner, let me try to add some value :)

The `--now` flag seems to only affect Pods, it does not propagate to Deployments/Services (according to https://github.com/kubernetes/kubernetes/issues/25055).